cnf-tests/README.md

```diff
@@ -96,7 +105,15 @@ To run only the configuration, and the latency test, you should provide the `gin
 the environment variable that contains the name of the performance profile that should be tested:
 
 ```bash
-docker run --rm -v $KUBECONFIG:/kubeconfig -e KUBECONFIG=/kubeconfig -e LATENCY_TEST_RUN=true -e LATENCY_TEST_RUNTIME=600 -e OSLAT_MAXIMUM_LATENCY=20 -e PERF_TEST_PROFILE=<performance_profile_name> quay.io/openshift-kni/cnf-tests /usr/bin/test-run.sh -ginkgo.focus="\[performance\]\[config\]|\[performance\]\ Latency\ Test"
+docker run --rm -v $KUBECONFIG:/kubeconfig \
+-e KUBECONFIG=/kubeconfig \
+-e LATENCY_TEST_RUN=true \
+-e LATENCY_TEST_RUNTIME=600 \
+-e OSLAT_MAXIMUM_LATENCY=20 \
+-e CYCLICTEST_MAXIMUM_LATENCY=20 \
+-e HWLATDETECT_MAXIMUM_LATENCY=20 \
```

Comment:
please add these new env vars as well as info on how to run to
https://github.com/openshift-kni/cnf-features-deploy/blob/master/cnf-tests/README.md#running-latency-test